### PR TITLE
dreamchecker

### PR DESCRIPTION
**Pull request by mikomyazaki**
Implements dreamchecker plus fixes all errors
See #7773

### CODE REVIEW

**Comment by mikomyazaki** [comment]
All working, so closing in favour of Skull's PR which he will hopefully merge this stuff into.

**Comment by mikomyazaki** [comment]
!review

**Comment by mikomyazaki** [comment]
I don't think this needs a changelog? Nothing player facin'.

File: code/modules/admin/verbs/mapping.dm [hunk 1]
```diff
@@ -126,9 +122,8 @@ var/intercom_range_display_status = 0
 					qdel(F)
 	feedback_add_details("admin_verb","mIRD") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
-var/list/debug_verbs = list (
-	/client/proc/do_not_use_these
```
Comment (Arrow768) [inline code/modules/admin/verbs/mapping.dm]: That exists so it´s also show in in the game that they should not be used in the game.

File: code/game/gamemodes/cult/ritual.dm [hunk 2]
```diff
@@ -0,0 +1,594 @@
+//This file was auto-corrected by findeclaration.exe on 25.5.2012 20:42:32
```
Comment (Arrow768) [inline code/game/gamemodes/cult/ritual.dm]: Where did that come from?

Comment (mikomyazaki) [reply code/game/gamemodes/cult/ritual.dm]: Not sure how that reappeared with the merge. Removed it again.

**Pull request merged**